storage/innobase/handler/ha_innodb.cc

```diff
@@ -11080,7 +11096,11 @@ create_table_info_t::create_table_def()
 
 		const char*	encrypt = m_create_info->encrypt_type.str;
 
-		if (!Encryption::is_none(encrypt)) {
+		const bool default_to_encrypt =
+			!Encryption::none_specified(encrypt)
```

Comment:
For reader very hard to tell the difference between "none_specified" and "is_none", consider renaming